desktop/cmp/form/inputs/Select.js

```diff
@@ -158,7 +164,7 @@ export class Select extends HoistInput {
                 menuPortalTarget: this.getOrCreatePortalDiv(),
 
                 inputId: props.id,
-                classNamePrefix: 'xh-select',
+                classNamePrefix: props.hideSelectedOptionCheck ? 'xh-select-no-check' : 'xh-select-default',
```

Comment:
Sure - I used that approach before considering how to apply a background when `hideSelectedOptionCheck` is enabled. Just a question of how to get at the `__option` div from the component... will look into react-select's api.

Reply:
Alternatively, perhaps you could add an additional cls to the overall component (at the top level) and use that to modify the handling of the nested option element.

Note that react-select has this `styles` prop that we are deliberately not using as there's no good way to merge possible app-level customizations in, and I can see apps needing/wanting to use it for custom styling.  So I would avoid that if we can.

Reply:
Updated - now adding an additional class to each menu div if checks have been disabled. There is still some padding logic in the css - hard to avoid without using react-select's style prop, but hopefully more readable now. 

I also kept in the custom box-shadow - minor difference but I think it looks better when zoomed in.